config/routes.rb

```diff
@@ -76,6 +76,7 @@
   get '/mentions-legales', to: 'pages#mentions_legales'
   get '/conditions-d-utilisation', to: 'pages#conditions_d_utilisation'
   get '/contact', to: 'pages#contact'
+  get '/statistiques', to: 'pages#statistics'
```

Comment:
```suggestion
  get '/statistiques', to: 'pages#statistiques'
```
question d'homogénéités